introduction/java_loops_i/Solution.java

```diff
@@ -0,0 +1,14 @@
+import java.util.Scanner;
+
+public class Solution {
+  static private final int MAX_NUMBER_OPERATIONS = 10;
```

Comment:
private static
The standard order is the following:
public/protected/private abstract static final